woocommerce/woocommerce-ios
# [WOO POS] Checkout totals update

## Description


- Updated some parts of the totals/checkout screen, added placeholder views etc.

## Steps to reproduce


## Testing information


- Enter POS
- Add few items in the cart, tap Checkout and try to do a fake cash payment (for now just changing states to update the UI)
- Remove items from the cart while being in checkout phase, check that you are brought back to cart building stage

## Screenshots


Cart creation and fake cash payment:

![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) - 2024-06-04 at 14 35 47](https://github.com/woocommerce/woocommerce-ios/assets/6242034/d2ddbca0-3327-42fc-94fe-7b05081e6d8c)

Emptying the cart and going back to cart building:

![Simulator Screen Recording - iPad Pro (11-inch) (4th generation) - 2024-06-04 at 14 35 58](https://github.com/woocommerce/woocommerce-ios/assets/6242034/ba7bf996-8042-4597-8eae-5604ee1b72fc)


---
